examples/AspNetCore/README.md

```diff
@@ -13,6 +13,32 @@ service name, version and the machine on which this program is running.
 The sample rate is set to emit all the traces using `AlwaysOnSampler`.
 You can try out different samplers like `TraceIdRatioBasedSampler`.
 
+## Export metrics to Prometheus
+
+Switch to Prometheus by setting `UseMetricsExporter` to `"prometheus"` in `appsettings.json`.
+
+Follow this [guide](../../docs/metrics/getting-started-prometheus-grafana/README.md#collect-metrics-using-prometheus)
+on how to set up Prometheus.
+
+Note that for this example, the `yml` file configuration needs an additional
+parameter `scheme` for HTTPS. Here goes an example:
+
+```yaml
+global:
+  scrape_interval: 10s
+  evaluation_interval: 10s
+scrape_configs:
+  - job_name: "otel"
+    scheme: https
+    static_configs:
+      - targets: ["localhost:53750"]
```

Comment:
where does 53750 come from?

Reply:
It is the randomly generated port number when running the project following the tutorial: https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&tabs=visual-studio#test-the-project.

I'll change it to a placeholder like `<port>` since it's a randomly generated number.